### ISSUE TITLE
[Feature Request] Add lambda_filter to datadog_integration_aws

### ISSUE THREAD

**Issue by kayman-mk** [DataDog/terraform-provider-datadog#1070]
Hi guys,

thanks for this Terraform provider!

I noticed that the `datadog_integration_aws` does not have the feature of "lambda_filter" to include certain lambdas only (like the filter_tag for EC2 instances). I am able to configure it via your [Website](https://app.datadoghq.eu/account/settings#integrations/amazon-web-services) but it would be nice to have this in Terraform as well.

Any chance to get an implementation for this?

Best regards
Matthias

![image](https://user-images.githubusercontent.com/26541330/118640150-ef449400-b7d8-11eb-9c14-8c275a75e7c0.png)

**Comment by therve** [comment]
Hi,

I believe you can achieve this using the tag filter resource: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_aws_tag_filter.

Let us know if that doesn't work for you. Thanks.

**Comment by kayman-mk** [comment]
Good point. I wasn't aware of this resource. Unfortunately I am an EU customer and the docs say it is for the US endpoint only.

Any other chance?

**Comment by skarimo** [comment]
Hi, 

The US only constraint has been removed for the `integration_aws_tag_filter` resource and you should be able to use it with EU region.
